string conn_name
string conn_type
string alias_name
bool latch
int throttle_rate
---
